src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ICompilationExtensions.cs

```diff
@@ -69,6 +69,12 @@ public static ImmutableArray<IAssemblySymbol> GetReferencedAssemblySymbols(this
         return builder.ToImmutableAndFree();
     }
 
+    public static INamedTypeSymbol? ArgumentExceptionType(this Compilation compilation)
+        => compilation.GetTypeByMetadataName(typeof(ArgumentException).FullName!);
```

Comment:
We should be using `GetBestTypeByMetadataName` for these, right?

Reply:
no clue.  none of hte rest of htese use that.  If so, open PR to update them to move to that if it's desired.